tools/install-powershell.ps1

```diff
@@ -271,12 +305,12 @@ try {
         $blobName = $metadata.BlobName
 
         # Get version from currently installed PowerShell Daily if available.
-        $pwshPath = if ($IsWinEnv) {Join-Path $Destination "pwsh.exe"} else {Join-Path $Destination "pwsh"}
-        $currentlyInstalledVersion = if(Test-Path $pwshPath) {
+        $pwshPath = if ($IsWinEnv) { Join-Path $Destination "pwsh.exe" } else { Join-Path $Destination "pwsh" }
```

Comment:
is there an expectation that this script runs down level? If not, I would suggest:
`$pwshPath = $IsWinEnv ? (join-path $destination pwsh.exe) : (join-path $destination pwsh)`